src/backend/distributed/executor/multi_utility.c

```diff
@@ -1035,6 +1121,43 @@ PlanAlterTableStmt(AlterTableStmt *alterTableStatement, const char *alterTableCo
 				constraint->skip_validation = true;
 			}
 		}
+#if (PG_VERSION_NUM >= 100000)
+		else if (alterTableType == AT_AttachPartition)
```

Comment:
Which part of the logic should be moved into separate function? It is very short logic, do you really wanto move it to another function?